main.cpp

```diff
@@ -50,91 +50,89 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
-
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+
+//  1)
+//  Noun:       computer
+//  action 1:   computer stores information
+computer.storeInformation();
+//  action 2:   computer retrieves information
+computer.retrieveInformation();
+//  action 3:   computer processes information
+computer.processInformation();
+//  2)
+//  Noun:       sun
+//  action 1:   the sun radiates light
+sun.radiateLight();
+//  action 2:   the sun radiates heat
+sun.radiateHeat();
+//  action 3:   the sun radiates solar energy
+sun.radiateSolarEnergy();
+//  3)
+//  Noun:       water
+//  action 1:   the water waves
+water.wave();
+//  action 2:   the water vaporizes
+water.vapourize();
+//  action 3:   the water freezes
+water.freeze();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       drake
+//  action 1:   drake raps
+drake.rap();
+//  action 2:   drake sings
+drake.sing();
+//  action 3:   drake dances
+drake.dance();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       head
+//  action 1:   the head nods
+head.nod();
+//  action 2:   the head rotates
+head.rotate();
+//  action 3:   the head bends
+head.bend();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       admin
+//  action 1:   adds user
+admin.addUser();
+//  action 2:   deletes user
+admin.deleteUser();
+//  action 3:   warns user
+admin.warnUser();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       god
+//  action 1:   god creates
+god.create();
+//  action 2:   god sustains
+god.sustain();
+//  action 3:   god cares
+god.care();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       devil
+//  action 1:   devil steals
+devil.steal();
+//  action 2:   devil kills
+devil.kill();
+//  action 3:   devil destroys
+devil.destroy();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       tesla
+//  action 1:   tesla walks
+tesla.walk();
+//  action 2:   tesla jumps
+tesla.jump();
+//  action 3:   tesla sings
+tesla.sing();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.